packages/cli/src/commands/hydrogen/init.ts

```diff
@@ -157,18 +169,21 @@ export async function runInit(
   let packageManager = await packageManagerUsedForCreating();
 
   if (packageManager !== 'unknown') {
-    const {installDeps} = await ui.prompt([
-      {
-        type: 'select',
-        name: 'installDeps',
-        message: `Install dependencies with ${packageManager}?`,
-        choices: [
-          {name: 'Yes', value: 'true'},
-          {name: 'No', value: 'false'},
-        ],
-        default: 'true',
-      },
-    ]);
+    const {installDeps} =
+      typeof options.installDeps === 'boolean'
+        ? {installDeps: String(options.installDeps)}
```

Comment:
I'd rather see the result of the prompt transformed to a boolean, rather than the other way around. There is a `result` function you can use to apply a transform after the user selects a choice when using `ui.prompt`.

Reply:
Actually, the `result` function is not working so I've changed it to something else :/